src/Generator.Bind/FuncProcessor.cs

```diff
@@ -358,7 +358,7 @@ private static string GetOverridesPath(string apiname, string apiversion, string
                 }
                 else if (enum_override.Value == "StringBuilder")
                 {
-                    type.QualifiedType = "StringBuilder";
+                    throw new Exception(apiname);
```

Comment:
Yup agree with this. Been a bit short on time to look at these again but will change these to better exceptions.